packages/scene-composer/src/components/panels/scene-components/AnchorComponentEditor.tsx

```diff
@@ -118,23 +123,25 @@ export const AnchorComponentEditor: React.FC<IAnchorComponentEditorProps> = ({
   );
 
   const iconOptions = useMemo(() => {
-    return Object.keys(SCENE_ICONS).map((sceneIcon) => ({
-      label: intl.formatMessage(i18nSceneIconsKeysStrings[sceneIcon]) || sceneIcon,
-      value: sceneIcon,
-    }));
+    return Object.keys(tagStyle ? SCENE_ICONS_CUSTOM : SCENE_ICONS).map((sceneIcon) => {
+      return {
+        label: intl.formatMessage(i18nSceneIconsKeysStrings[sceneIcon]) || sceneIcon,
+        value: sceneIcon,
+      };
+    });
   }, []);
 
   const iconSelectedOptionIndex = useMemo(() => {
     if (!anchorComponent.icon) {
       return -1;
     }
 
-    const sceneResourceInfo = getSceneResourceInfo(anchorComponent.icon);
-    return Object.keys(SCENE_ICONS).indexOf(sceneResourceInfo.value);
+    const sceneResourceInfo = getSceneResourceInfo(anchorComponent.icon, tagStyle);
+    return Object.keys(tagStyle ? SCENE_ICONS_CUSTOM : SCENE_ICONS).indexOf(sceneResourceInfo.value);
```

Comment:
same here, just use `SCENE_ICONS` which has Custom, and display based on selected resource

Reply:
I did not follow your comment.

Reply:
basically no change needed here, just add Custom to `SCENE_ICONS` and use `SCENE_ICONS` here